recipes/SimpleElastix/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Tell the build to use conda's Python
+SET PYTHON_EXECUTABLE=$PYTHON
+SET PYTHON_INCLUDE_DIR=$PREFIX/include
+SET PYTHON_LIBRARY=$PREFIX/lib
+
+# Build
+mkdir build
+cd build
+cmake ../SimpleElastix/SuperBuild
+make -j4
```

Comment:
Yeah, I have been pushing people to not use `-j` at all. Part of the issue is some CIs are not great about informing you how many CPUs you actually have versus what kind of node you are on. So, it may return something like 16, but you are sharing this with other jobs.